pages/branchMerge.js

```diff
@@ -0,0 +1,333 @@
+        },
+      )
+      /* Sample return value
+      {
+          "mergeNeeded": false,
+          "conflict": false,
+          "success": true,
+          "userBranchDeleted": false,
+          "error": false,
+          "message": "no merge needed"
+      }
+      */
+
+      if ( _results.error ) {
+        setPageStatus(`Update of user branch failed:\n\n${_results.message}`)
+        console.log("Update from master failed:\n",_results.message)
+      } else {
+        setPageStatus(`Update of user branch succeeded!`)
+      }
+      setDoUpdate(false)
+      setBranch('')
+    }
+
+
+    if ( doUpdate ) {
+      doUpdateUserBranch()
+    }
+  }, [doUpdate, branch, server, organization, languageId, resourceId, authentication])
+
+  useEffect( () => {
+    async function doMergeUserBranch() {
+      const _rid = resourceIdMapper(organization,resourceId)
+      const tokenid = authentication.token.sha1
+      const repo = languageId + '_' + _rid
+
+      console.log('doMergeUserBranch()')
+      const _results = await mergeUserIntoDefaultBranch(
+        {
+          server, owner: organization, repo, userBranch: branch, tokenid,
+        },
+      )
+      /* Sample return value
+      {
+          "mergeNeeded": false,
+          "conflict": false,
+          "success": true,
+          "userBranchDeleted": false,
+          "error": false,
+          "message": "no merge needed"
+      }
+      */
+
+      if ( _results.error ) {
+        setPageStatus(`Merge of user branch into master failed:\n\n${_results.message}`)
+        console.log("Merge into master failed:\n",_results.message)
+      } else {
+        setPageStatus(`Merge of user branch into master succeeded!`)
+      }
+      setDoMerge(false)
+      setBranch('')
+      setTimeout(
+        () => setPageStatus(pageDefaultStatusMessage),
+        10000,
+      )
+    }
+
+
+    if ( doMerge ) {
+      doMergeUserBranch()
+    }
+  }, [doMerge, branch, server, organization, languageId, resourceId, authentication])
+
+  return (
+    <>
+      <Layout>
+        <Paper>
+          <div className='flex flex-col justify-center items-center'>
+            <div className='flex flex-col w-full px-4 lg:w-132 lg:p-0'>
+              <h1 className='mx-4'>No Conflict Merge Management</h1>
+              <FormControl>
+                <FormLabel id="resource-type-radio-buttons-group-label" component="legend">Select a Resource</FormLabel>
+                <RadioGroup
+                  aria-labelledby="resource-type-radio-buttons-group-label"
+                  // defaultValue="lt"
+                  name="resource-type-radio-buttons-group"
+                  row
+                  onChange={handleResourceChange}
+                >
+                  {resources.map( ({ id,name }) =>
+                    <FormControlLabel key={id} value={id} control={<Radio />} label={name} />,
+                  )}
+                </RadioGroup>
+              </FormControl>
+              <Autocomplete
```

Comment:
Can you have this say "Select a Branch"?

![image](https://user-images.githubusercontent.com/2839925/229152378-9c17e608-4f2b-477a-a584-2b6340a1e40b.png)

Reply:
Also, are we going to tell them this branch has conflicts, can't do anything, or just that there are no changes?

Reply:
Both comments are resolved with most recent pushes.